### ISSUE TITLE
eGon data fails after fresh install due to package problems: `psycopg2.errors.UndefinedTable`

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#641]
I made a fresh install on our server and eGon-data does not start with a fresh venv. However, my previous venv worked correctly.

I created a diff of pip freeze:
[pip_diff.txt](https://github.com/openego/eGon-data/files/7888856/pip_diff.txt)

Error message:
```
(venv) CENSORED@ws-02lin:/storage/eGon-data2/working-dir$ egon-data --dataset-boundary Everything --database-port 59779 --jobs 32 --docker-container-name egon-data2-ja --compose-project-name egon-data2-ja --airflow-port 
8082 serve                                                                                                                                                                                                                                   
Building egon-data-local-database-service                                                                                                                                                                                                    
Step 1/5 : FROM postgres:12                                                                                                                                                                                                                  
 ---> 91d892a67245                                                                                                                                                                                                                           
Step 2/5 : RUN apt-get update                                                                                                                                                                                                                
 ---> Using cache                                                                                                                                                                                                                            
 ---> 3f142b0dcff3                                                                                                                                                                                                                           
Step 3/5 : RUN apt-get install -y postgresql-12-postgis-3                                                                                                                                                                                    
 ---> Using cache                                                                                                                                                                                                                            
 ---> 457f8093db13                                                                                                                                                                                                                           
Step 4/5 : RUN apt-get update                                                                                                                                                                                                                
 ---> Using cache                                                                                                                                                                                                                            
 ---> 0a83449b75b8                                                                                                                                                                                                                           
Step 5/5 : RUN apt-get install -y postgresql-12-pgrouting                                                                                                                                                                                    
 ---> Using cache                                                                                                                                                                                                                            
 ---> d9babf886905                                                                                                                                                                                                                           
Successfully built d9babf886905                                                                                                                                                                                                              
Successfully tagged postgres:12-postgis 
/storage/eGon-data2/venv/lib/python3.8/site-packages/airflow/configuration.py:762: DeprecationWarning: You have two airflow.cfg files: CENSORED_HOME/airflow/airflow.cfg and /storage/eGon-data2/working-dir/airflow/
airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /storage/eGon-data2/working-dir/airflow/airflow.cfg, and you should remove the other file        
  warnings.warn(                                                                                                                                                                                                                             
/storage/eGon-data2/venv/lib/python3.8/site-packages/airflow/configuration.py:762: DeprecationWarning: You have two airflow.cfg files: CENSORED_HOME/airflow/airflow.cfg and /storage/eGon-data2/working-dir/airflow/
airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /storage/eGon-data2/working-dir/airflow/airflow.cfg, and you should remove the other file        
  warnings.warn(                                                                                                                                                                                                                             
Traceback (most recent call last):                                                                                                                                                                                                           
  File "/storage/eGon-data2/venv/bin/airflow", line 26, in <module>                                                                                                                                                                          
    from airflow.bin.cli import CLIFactory                                                                                                                                                                                                   
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/airflow/bin/cli.py", line 82, in <module>                                                                                                                                       
    from airflow.www.app import (cached_app, create_app)                                                                                                                                                                                     
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/airflow/www/app.py", line 42, in <module>                                                                                                                                       
    from airflow.www.blueprints import routes                                                                                                                                                                                                
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/airflow/www/blueprints.py", line 25, in <module>                                                                                                                                
    from airflow.www import utils as wwwutils                                                                                                                                                                                                
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/airflow/www/utils.py", line 36, in <module>                                                                                                                                     
    from wtforms.compat import text_type                                                                                                                                                                                                     
ModuleNotFoundError: No module named 'wtforms.compat'                                                                                                                                                                                        
Traceback (most recent call last):                                                                                                                                                                                                           
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context                                                                                                                      
    self.dialect.do_execute(                                                                                                                                                                                                                 
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute                                                                                                                          
    cursor.execute(statement, parameters)                                                                                                                                                                                                    
psycopg2.errors.UndefinedTable: relation "connection" does not exist                                                                                                                                                                         
LINE 2: FROM connection                                                                                                                                                                                                                      
             ^                                                                                                                                                                                                                               
                                                                                                                                                                                                                                             
                                                                                                                                                                                                                                             
The above exception was the direct cause of the following exception:                                                                                                                                                                         
                                                                                                                                                                                                                                             
Traceback (most recent call last):                                                                                                                                                                                                           
  File "/storage/eGon-data2/venv/bin/egon-data", line 33, in <module>                                                                                                                                                                        
    sys.exit(load_entry_point('egon.data', 'console_scripts', 'egon-data')())                                                                                                                                                                
  File "/storage/eGon-data2/code/src/egon/data/cli.py", line 443, in main                                                                                                                                                                    
    egon_data.main(sys.argv[1:])                                                                                                                                                                                                             
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/click/core.py", line 782, in main                                                                                                                                               
    rv = self.invoke(ctx)
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/click/core.py", line 1256, in invoke                                                                                                                                            
    Command.invoke(self, ctx)                                                                                                                                                                                                                
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/click/core.py", line 1066, in invoke                                                                                                                                            
    return ctx.invoke(self.callback, **ctx.params)                                                                                                                                                                                           
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/click/core.py", line 610, in invoke                                                                                                                                             
    return callback(*args, **kwargs)                                                                                                                                                                                                         
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/click/decorators.py", line 21, in new_func                                                                                                                                      
    return f(get_current_context(), *args, **kwargs)                                                                                                                                                                                         
  File "/storage/eGon-data2/code/src/egon/data/cli.py", line 382, in egon_data                                                                                                                                                               
    airflow.query(Connection).filter_by(conn_id="egon_data").one_or_none()                                                                                                                                                                   
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3459, in one_or_none                                                                                                                             
    ret = list(self)                                                                                                                                                                                                                         
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3535, in __iter__                                                                                                                                
    return self._execute_and_instances(context)                                                                                                                                                                                              
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3560, in _execute_and_instances                                                                                                                  
    result = conn.execute(querycontext.statement, self._params)                                                                                                                                                                              
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute                                                                                                                               
    return meth(self, multiparams, params)                                                                                                                                                                                                   
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection                                                                                                                
    return connection._execute_clauseelement(self, multiparams, params)                                                                                                                                                                      
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement                                                                                                                
    ret = self._execute_context(                                                                                                                                                                                                             
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context                                                                                                                      
    self._handle_dbapi_exception(                                                                                                                                                                                                            
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception                                                                                                               
    util.raise_(                                                                                                                                                                                                                             
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_                                                                                                                                 
    raise exception                                                                                                                                                                                                                          
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context                                                                                                                      
    self.dialect.do_execute(                                                                                                                                                                                                                 
  File "/storage/eGon-data2/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute                                                                                                                          
    cursor.execute(statement, parameters)                                                                                                                                                                                                    
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "connection" does not exist                                                                                                                                       
LINE 2: FROM connection                                                                                                                                                                                                                      
             ^                                                                                                                                                                                                                               
                                                                                                                                                                                                                                             
[SQL: SELECT connection.password AS connection_password, connection.extra AS connection_extra, connection.id AS connection_id, connection.conn_id AS connection_conn_id, connection.conn_type AS connection_conn_type, connection.h[304/1674]
nection_host, connection.schema AS connection_schema, connection.login AS connection_login, connection.port AS connection_port, connection.is_encrypted AS connection_is_encrypted, connection.is_extra_encrypted AS connection_is_extra_encr
ypted                                                                                                                                                                                                                                        
FROM connection                                                                                                                                                                                                                              
WHERE connection.conn_id = %(conn_id_1)s]                                                                                                                                                                                                    
[parameters: {'conn_id_1': 'egon_data'}]                                                                                                                                                                                                     
(Background on this error at: http://sqlalche.me/e/13/f405)
```

Related: #343 #556

**Comment by gnn** [comment]
So, I did some research and the issue is related to us still not having upgraded to `"airflow>=2.0"`. The `airflow` version we're currently using pulls in the `WTForms` package indirectly via `Flask-WTF`. The latter does not specify an upper version limit on `WTForms`, so the most recent one gets installed. But `airflow` imports `wtforms.compat` which got removed when `WTForms` bumped its major version to three. So a quick fix would be to put a `"wtforms<3"` into "setup.py".
BUT things are a tiny bit more complicated. We have a test which would've caught this issue, but it currently gets skipped because it needs Docker and/or PostgreSQL and that's not yet configured for GH actions. So please give me another day to create a feature branch on which the test gets enabled again off of "dev" BEFORE I'm going to merge the above fix into "dev". In the meantime, feel free to figure out whether this fix is enough or whether it leaves other errors occurring down the line.
Also, I need to figure out why this wasn't caught during any of the weekend runs...

**Comment by gnn** [comment]
PR #692 should fix this. Would you mind testing whether this is correct?
I also know now why this wasn't caught during the weekend runs and this issue is fixed as well. Work and getting Docker and/or PostgreSQL into GH actions to catch stuff like this earlier is also progressing, so if the fix works, the issue can be closed.

**Comment by nesnoj** [comment]
Has been solved by #692
